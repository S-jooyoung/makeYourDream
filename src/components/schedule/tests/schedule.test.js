// import dependencies
import React from "react";

import Schedule from "../schedule";
import renderer from "react-test-renderer";

describe("Schedule", () => {
  let ScheduleCompoent;
  const title = "testTitle";
  const info = "testInfo";

  beforeEach(() => {
    ScheduleCompoent = <Schedule title={title} info={info} />;
  });

  it("render", () => {
    const component = renderer.create(ScheduleCompoent).toJSON();

    expect(component).toMatchSnapshot();
  });
});
