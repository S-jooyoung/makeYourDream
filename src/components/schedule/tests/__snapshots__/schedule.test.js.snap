// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Schedule render 1`] = `
Array [
  <h1
    className="title"
  >
    testTitle
  </h1>,
  <p
    className="info"
  >
    testInfo
  </p>,
]
`;
